.github/dependabot.yml

```diff
@@ -73,3 +73,8 @@ updates:
     - dependency-name: org.osgi:org.osgi.service.log
       versions:
         - ">= 1.4.0"
+
+    # See https://github.com/killbill/killbill-oss-parent/pull/654
+    - dependency-name: jakarta.servlet:jakarta.servlet-api
```

Comment:
Does this tell `dependabot` to not do anything for versions of this package > `5.0.0` - we do seem to have version [4.0.4 configured](https://github.com/killbill/killbill-oss-parent/blob/master/pom.xml#L120). Not following ... 😭

Reply:
:sweat_smile: It is mean "do not create PR for servlet >= 5.0.0" . If, in the future there's servlet 4.0.5 or 4.1.x released, dependabot will still create PR for us. CMIIW @pierre